.github/actions/setup-node/action.yml

```diff
@@ -22,6 +26,12 @@ runs:
       with:
         version: 7.12.1
 
+    - name: Make sure pnpm always has a cache
+      if: ${{ inputs.enable-pnpm-cache == 'true' }}
+      shell: bash
+      run: |
+        mkdir -p `pnpm store path`
```

Comment:
I'm faithfully adding this in here to keep parity with what the workflow job does now, but not sure if it works, and if it should be applied unviersally to all other invocations of setup-node. Not _really_ trying to go down that path in this PR.

Reply:
There was an issue with the pnpm or setup-node action that would throw an error at this step in _some_ cases, I think around the pnpm 6=>7 transition.

I haven't gone back to check on it; this was the official workaround at that point in time.

This is safe to do and is at most a no-op.